content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -40,7 +40,8 @@ By default if no kubelet network plugin is specified, the `noop` plugin is used,
 
 The CNI plugin is selected by passing Kubelet the `--network-plugin=cni` command-line option.  Kubelet reads a file from `--cni-conf-dir` (default `/etc/cni/net.d`) and uses the CNI configuration from that file to set up each pod's network.  The CNI configuration file must match the [CNI specification](https://github.com/containernetworking/cni/blob/master/SPEC.md#network-configuration), and any required CNI plugins referenced by the configuration must be present in `--cni-bin-dir` (default `/opt/cni/bin`).
 
-If there are multiple CNI configuration files in the directory, the first one in lexicographic order of file name is used.
+If there are multiple CNI configuration files in the directory, the first by name in lexicographicorder is used.
```

Comment:
missing space in "lexicographic order"

Reply:
Here's a suggested reword
```suggestion
If there are multiple CNI configuration files in the directory, the kubelet uses the configuration file that comes first by name in lexicographic order.
```

Reply:
@miteshskj Thanks that was hopeless of me
@sftim That's actually much better